Cargo.toml

```diff
@@ -52,7 +52,8 @@ derive_more = "0.99"
 environmental = { version = "1.1.4", default-features = false }
 ethereum = { version = "0.15.0", default-features = false }
 ethereum-types = { version = "0.14.1", default-features = false }
-evm = { version = "0.41.1", default-features = false }
+#evm = { version = "0.41.1", default-features = false }
+evm = { git = "https://github.com/moondance-labs/evm", branch = "agustin-contract-deploy-filter", default-features = false }
```

Comment:
TODO: change the pointed branch once https://github.com/moondance-labs/evm/pull/1 is merged.